config/initializers/bootstrap_field_errors.rb

```diff
@@ -4,15 +4,15 @@
 # wrapping them in a div that breaks bootstrap layouts
 ActionView::Base.field_error_proc = proc do |html_tag, instance|
   feedback = ""
-  if html_tag.match(/^<(input|textarea|select)/) && instance.error_message.present?
+  if html_tag.match(/^<(input|textarea|select|trix-editor)/) && instance.error_message.present?
     msg = instance.error_message.join(", ").capitalize
     feedback = "<div class=\"invalid-feedback\">#{msg}</div>"
   end
 
   if html_tag.match?(/class="(.*?)"/)
     (html_tag.sub(/class="(.*?)"/, 'class="\1 is-invalid"') + feedback).html_safe
   else
-    (html_tag.sub(/(\/>|>)/, 'class="is-invalid" \1') + feedback).html_safe
```

Comment:
The previous regex was tripping up on `data-action="whatev->something"` stimulus actions.  So it would insert this invalid-feedback div right in the middle of the data attributes.

Reply:
better!